doc/nrf/libraries/modem/lte_lc.rst

```diff
@@ -87,17 +80,67 @@ The following block of code shows how you can use the API to establish an LTE co
    }
 
 The code block demonstrates how you can use the library to asynchronously set up an LTE connection.
-For more information on the callback events received in :c:type:`lte_lc_evt_handler_t` and data associated with each event, see the documentation on :c:struct:`lte_lc_evt`.
 
-The following list mentions some of the information that can be extracted from the received callback events:
+Additionally, to enable specific functionalities and receive specific events from the library, you must enable the corresponding modules through their respective Kconfig options:
+
+Connection Parameters Evaluation:
+  Enable the :kconfig:option:`CONFIG_LTE_LC_CONN_EVAL` Kconfig option to enable all the following functionalities related to Connection Parameters Evaluation:
+
+  * :c:func:`lte_lc_conn_eval_params_get`
+
+eDRX (Extended Discontinuous Reception):
+  Enable the :kconfig:option:`CONFIG_LTE_LC_EDRX` Kconfig option to enable all the following functionalities related to eDRX:
+
+  * :c:enumerator:`LTE_LC_EVT_EDRX_UPDATE` events
+  * :c:func:`lte_lc_ptw_set`
+  * :c:func:`lte_lc_edrx_param_set`
+  * :c:func:`lte_lc_edrx_req`
+  * :c:func:`lte_lc_edrx_get`
```

Comment:
Agree. On a side note, we definitely need to bring some consistency in the Kconfig names in the future.

Reply:
I'd probably list in the documentation all Kconfig options for a given module because of the inconsistent naming that we have to make things clear.

I haven't renamed the Kconfig's to avoid a breaking change because we do them too much already within NCS. It has been in my mind many times though.